hash.rb

```diff
@@ -1,39 +1,39 @@
+# :markup: markdown
```

Comment:
Is there a reason why we're using markdown and not RDoc?

Reply:
@st0012 believes (I think) that markdown is better going forward;  personally, I'm agnostic.  I'll be happy to switch if that's best.

@jeremyevans (and any other interested parties), should we say anything in the doc guide about choosing a markup format?

Reply:
I would prefer continuing to document in rdoc format, unless there are major benefits from using markdown for what is being documented.  If we do switch from rdoc to markdown, that will be done by a tool, with all changes being made at the same time.  After that point, we would write markdown.

Reply:
Sorry that I think the confusion came from when @BurdetteLamar was rewriting IRB's documentation, I said my preference was Markdown. I didn't make it clear that it is only the case inside IRB based on the maintainers' preferences.

As for `ruby/ruby`, my personal opinion is:
-  For page documents, like contributing guides and maintainers list...etc., switching is beneficial as `.md` files are way easier to review on GH and/or preview locally through editor extensions. But there isn't any guideline on this yet AFAIK.
- For comment documentation, the benefit is less obvious as we review on the raw markup anyway. And in projects like `ruby/ruby`, keeping the markup consistent across API documentation is probably more beneficial for now.

Reply:
Restored to default markup (rdoc).